src/components/create_client/EditMobileClientBaseClass.js

```diff
@@ -62,8 +62,8 @@ class EditMobileClientBaseClass extends Component {
         <Grid bsClass="create-client-form">
           <Form vertical="true">{generatedFields}</Form>
         </Grid>
-        <p>JavaScript-based mobile apps can be configured for a variety of mobile platforms.</p>
-        <p>Our JavaScript SDK supports the following frameworks:</p>
+        <p>Create cross-platform mobile apps using JavaScript.</p>
+        <p>You can use the following JavaScript framesorks:</p>
```

Comment:
This does not match the description/verification steps:

Replace 'Our JavaScript SDK supports the following frameworks' with 'Our JavaScript SDK can be used with the following frameworks'